packages/framework/src/Model/Product/ProductOnCurrentDomainElasticFacade.php

```diff
@@ -86,9 +85,9 @@ public function __construct(
         ProductAccessoryRepository $productAccessoryRepository,
         ProductElasticsearchRepository $productElasticsearchRepository,
         ProductFilterCountDataElasticsearchRepository $productFilterCountDataElasticsearchRepository,
-        ProductFilterDataToQueryTransformer $productFilterDataToQueryTransformer,
+        ?ProductFilterDataToQueryTransformer $productFilterDataToQueryTransformer,
```

Comment:
This is interesting, do I get it right it's because this dependency will be deprecated and it's without a default value because it's in the middle of arguments?
I wonder, is it necessary to make it nullable? Isn't marking as deprecated enough?

Reply:
fixed in c187ecbe1cfdaa375e049ffd32a84821d459eadb